boot/zephyr/Kconfig

```diff
@@ -289,6 +289,26 @@ config BOOT_ENCRYPT_X25519
 	  described under "ECIES-X25519 encryption" in docs/encrypted_images.md.
 endif # !SINGLE_APPLICATION_SLOT
 
+config BOOT_ENCRYPT_KEY_FILE
+	string "PEM key file"
+	default ""
```

Comment:
When the keys are auto generated, ideally there would be no need for default keys in boot/zephyr/keys.c.
Those could be removed like it was done for the signature keys. Specifying enc-ec256-priv.pem, enc-rsa2048-priv.pem, enc-x25519-priv.pem as default keys for the related encryption mechanisms would be a first step.
For X25519 you would additionally have to take care that the automatically generated name does not match.
As result the BOOT_USE_ENCRYPT_KEY_FILE would no longer be needed.

Reply:
Adopted the same strategy as for generating public signature keys.